tests/test_utilities.py

```diff
@@ -311,7 +313,9 @@ def test_get_whitelisted_serialized_skills(self):
             } for expected_skill in expected_skills
         ]
 
-        actual_serialized_skills = utils.get_whitelisted_serialized_skills(course_key=COURSE_KEY)
+        actual_serialized_skills = utils.get_whitelisted_serialized_skills(
+            key_or_uuid=COURSE_KEY, product_type='Course'
```

Comment:
nit: can we move product_types to a constant to avoid any typo error during its usage.

Something like this
PRODUCT_TYPE= {'course': 'Course', 'program': 'Program'}

Reply:
well in that case, refactoring around taking a decision about `Program` or `Course` may be possible but argument passing will remain the same. 🤔

Reply:
We can also use Enum, and can pass them as argument, 
for example:
https://github.com/openedx/course-discovery/blob/master/course_discovery/apps/course_metadata/constants.py#L17

Reply:
thanks for the suggestion, I am considering this refactoring in my subsequent PR. 👍